### PR TITLE
'namespace/lookup' now returns null on 404 instead of throwing

### PR DESCRIPTION
**Pull request by taoeffect**

### CODE REVIEW

File: frontend/views/components/CreateGroupSteps/GroupInvitees.vue
Comment (sandrina-p) [inline frontend/views/components/CreateGroupSteps/GroupInvitees.vue:48]: This file isn't being used at all, can be deleted. Invites happen at `AddMembers.vue`

**Review by sandrina-p — APPROVED** [review]

**Comment by taoeffect** [comment]
**NOTE: this is an exceptional merge that violates Rule 4 of our contributions guidelines because it's during a hackathon and I have approval from @sandrina-p to merge it, and @sandrina-p cannot merge it herself because the build is failing, but it's an error unrelated to this PR.**

File: frontend/views/components/CreateGroupSteps/GroupInvitees.vue
Comment (taoeffect) [reply frontend/views/components/CreateGroupSteps/GroupInvitees.vue:48]: Makes sense to delete it in your PR :) 👍

**Pull request merged**
